handbook/data-stores/data-stores.md

```diff
@@ -0,0 +1,89 @@
+# MongoDB
+
+MongoDB is used by the majority of SystemLink services as the primary datastore. MongoDB Community Edition is included in the default installation of SystemLink will launch when the SystemLink server boots.
+
+!!!note
+    By default SystemLink stores all data on databases and the file system hosted on the SystemLink application server. This is referred to as a *single node* deployment. To improve the reliability of data storage and scale to larger volumes of data, remote data stores are recommended for all production deployments. This is referred to as a *multi node* deployment.
+
+## Assumptions and Prerequisites
+
```

Comment:
Can give this a bit more structure and help the flow if we put a short sentence here introducing this list. Something like 'Before you configure a remote MongoDB to use with SystemLink, ensure you have the following:'

Reply:
Great feedback. I will add this as a lead in